tests/fast-integration/skr-test/skr-test.js

```diff
@@ -68,6 +79,27 @@ describe("SKR test", function() {
   it("order.created.v1 event should trigger the lastorder function", async function () {
     await sendEventAndCheckResponse();
   });
+
+  it("Deletes the resources that have been created", async function () {
+    await deleteMockTestFixture("mocks", testNS);
+  });
+
+  // Check audit log for AWS
+  if (process.env.KEB_PLAN_ID == "361c511f-f939-4621-b228-d0fb79a1fe15") {
```

Comment:
Please extract this UUID to some meaningful const, for example `AWS_PLAN_ID`
```
  const AWS_PLAN_ID = "361c511f-f939-4621-b228-d0fb79a1fe15";
  ...
  if (process.env.KEB_PLAN_ID === AWS_PLAN_ID) {
```

Reply:
fixed